backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -304,32 +288,32 @@ public String getOrderingFacilityCounty() {
 
 	@JsonProperty("Ordering_facility_name")
 	public String getOrderingFacilityName() {
-		return org.getFacilityName();
+		return facility.getFacilityName();
 	}
 
 	@JsonProperty("Ordering_facility_phone_number")
 	public String getOrderingFacilityPhoneNumber() {
-		return "5202475313";
+		return facility.getTelephone();
```

Comment:
@timothybest-usds should I revert the changes to this file so we can just merge this without breaking the export?

Reply:
no worries about the export. next one isn't until 9 and I can add them manually, update the facility, or preferred merge in the edit facility information PR (should be rebased on this branch and working by end of day)

Reply:
We should coordinate offline on that last bit, I had started some schema iteration that might profitably mix in here—but otherwise cool, I shall fire away!